client-sdk/go/examples/basicauth_client.go

```diff
@@ -108,4 +109,34 @@ func main() {
 	// logger.Info("Result", "result", res) //This gives the result in json format
 	// logger.Info("HTTP status", "status", resp.Status)
 	//=======================================================
+
+	//======List orphan volumes======
```

Comment:
I see that some of the examples are commented out while some are not. Can you describe the intent of this?

Reply:
Say if you just want to test invocation of a single API, it's easier to uncomment just that API example and execute the code. Do you want to structure it in a different way?

Reply:
Here's an example of invoking a single endpoint after adjusting the basicauth credentials and server url.

```
agohilZLVDQ:examples agohil$ go run basicauth_client.go 
2023-05-11T16:24:37.109-0700    INFO    Main    examples/basicauth_client.go:123        Result  {"result": {"totalOrphans":0,"orphanVolumes":[],"retryAfterMinutes":8}}
2023-05-11T16:24:37.109-0700    INFO    Main    examples/basicauth_client.go:124        HTTP status    {"status": "200 OK"}
```

The users can copy whatever example they want to incorporate in their code.

Reply:
I see. Makes sense.